### PR TITLE
Add coinbase amount and account creation fee to GraphQL

### PR DESCRIPTION
**Pull request by mrmr1993**
This PR adds the `protocolAmounts` field to the GraphQL API, containing the coinbase amount and account creation fee.

Checklist:

- [ ] Document code purpose, how to use it
  + Mention expected invariants, implicit constraints
- [ ] Tests were added for the new behavior
  + Document test purpose, significance of failures
  + Test names should reflect their purpose
- [ ] All tests pass (CI will check this if you didn't)
- [ ] Serialized types are in stable-versioned modules
- [ ] Does this close issues? List them:

### CODE REVIEW

File: src/lib/coda_graphql/coda_graphql.ml [hunk 1]
```diff
@@ -431,6 +431,22 @@ module Types = struct
       ~doc:"Status for whenever the blockchain is reorganized"
       ~values:[enum_value "CHANGED" ~value:`Changed]
 
+  let protocol_amounts =
+    obj "ProtocolAmounts" ~fields:(fun _ ->
```
Comment (bkase) [inline src/lib/coda_graphql/coda_graphql.ml:435]: Can you rename to `GenesisConstants` for future proofing

File: src/lib/coda_graphql/coda_graphql.ml [hunk 2]
```diff
@@ -431,6 +431,22 @@ module Types = struct
...
+        [ field "accountCreationFee" ~typ:(non_null uint64)
+            ~doc:"The fee charged to create a new account"
+            ~args:Arg.[]
+            ~resolve:(fun {ctx= coda; _} () ->
+              (Coda_lib.config coda).precomputed_values.constraint_constants
+                .account_creation_fee |> Currency.Fee.to_uint64 )
+        ; field "coinbaseReward" ~typ:(non_null uint64)
```
Comment (bkase) [inline src/lib/coda_graphql/coda_graphql.ml:442]: nit: Call it `coinbase`

File: src/lib/coda_graphql/coda_graphql.ml [hunk 3]
```diff
@@ -2594,6 +2610,13 @@ module Queries = struct
         | None ->
             [] )
 
+  let protocol_amounts =
+    field "protocolAmounts"
```
Comment (bkase) [inline src/lib/coda_graphql/coda_graphql.ml:2614]: Can you rename to `genesisConstants` for future proofing

File: src/lib/coda_graphql/coda_graphql.ml [hunk 4]
```diff
@@ -2594,6 +2610,13 @@ module Queries = struct
...
+      ~doc:"The currency amounts for different events in the protocol"
```
Comment (bkase) [inline src/lib/coda_graphql/coda_graphql.ml:2615]: Doc: "The constants used to determine the configuration of the genesis block and all of its transitive dependencies" (something like that)

File: src/lib/coda_graphql/coda_graphql.ml (hunk 1 above)
Comment (psteckler) [reply src/lib/coda_graphql/coda_graphql.ml:435]: In the code that produces these values, the name is `constraint constants`.  Shouldn't there be one name, everywhere?

Comment (mrmr1993) [reply src/lib/coda_graphql/coda_graphql.ml]: The type is `Genesis_constants.Constraint_constants.t`, conceptually supposed to be the part of the genesis constants where modifications will affect the snark constraint system. This is useful for knowing what can be modified (eg. through the runtime config), but I don't think we need that level of granularity here.

Comment (psteckler) [reply src/lib/coda_graphql/coda_graphql.ml]: :+1:

**Pull request merged**